subcommands/devices/show.go

```diff
@@ -35,6 +35,7 @@ func doShow(cmd *cobra.Command, args []string) {
 	device := getDevice(cmd, args[0])
 
 	fmt.Printf("UUID:\t\t%s\n", device.Uuid)
+	fmt.Printf("Name:\t\t%s\n", device.Name)
```

Comment:
This looks like a separate functional change.
It clearly belongs to this PR, and I get why it is here...
...it still deserves a separate commit with a clear message.

Reply:
I was a little on the fence there also. It wasn't necessary(possible redundant) until the addition of the `--by-uuid` flag. i'll split